src/Microsoft.DotNet.VersionTools/Dependencies/IDependencyUpdater.cs

```diff
@@ -14,6 +14,6 @@ public interface IDependencyUpdater
         /// <summary>
         /// Updates based on the given build infos and returns build infos used during update.
         /// </summary>
-        IEnumerable<BuildInfo> Update(IEnumerable<BuildInfo> buildInfos);
+        IEnumerable<DependencyUpdateTask> Update(IEnumerable<BuildInfo> buildInfos);
```

Comment:
Yeah, I called this `GetUpdateTasks` on `DependencyUpdater` so I'll rename this also to `GetUpdateTasks`. Thanks.